extension/intents/tabs/tabs.js

```diff
@@ -597,3 +598,166 @@ intentRunner.registerIntent({
     context.displayText(message);
   },
 });
+
+intentRunner.registerIntent({
+  name: "tabs.selectAllTabs",
+  async run(context) {
+    const activeTab = await context.activeTab();
+    const tabs = await browser.tabs.query({ currentWindow: true});
+    const allWindows = await browser.windows.getAll({
+      windowTypes: ["normal"],
+    });
+
+    const matchingTabs = await getMatchingTabs({
+      tabs,
+      activeTab,
+      sort_by_index: true,
+      allWindows: context.parameters.allWindows === "false"
+    });
+
+    if (matchingTabs.length === 0) {
+      const exc = new Error("No tab that matches the query");
+      exc.displayMessage = "There is no tab that matches the query";
+      throw exc;
+    }
+
+    await selectAllTabs(matchingTabs);
+
+  },
+});
+
+async function selectAllTabs(tabs){
+  if(tabs.length !== 0){
+    browser.tabs.query({
+      currentWindow: true
+    }).then((tabs) => {
+      for (var tab of tabs) {
+        if (!tab.active){
+          await browser.tabs.update(tabId, {
+              highlighted: true,
+              active: false,
```

Comment:
Does this need to specify `active` at all?

Reply:
![Capture111](https://user-images.githubusercontent.com/22684019/78754558-50bef000-7995-11ea-854e-62953684ad37.PNG)

Highlight : (Adds or removes the tab from the current selection. If true and the tab is not highlighted, it will become active by default.

If you only want to highlight the tab without activating it, Firefox accepts setting highlighted to true and active to false. Other browsers may activate the tab even in this case.)


Actually, we don't have to activate all the tabs , but only highlight them. So, whatever tab is active will be active other tabs wont be active only highlighted.
So, I think it is needed